package.json

```diff
@@ -48,7 +48,9 @@
     "screenshots": "node scripts/screenshots.js --screenshot-dir ./screenshots --styleguide-dir ./build/react",
     "test": "yarn test:jest",
     "test:jest": "jest --verbose",
-    "transpile": "env BABEL_ENV=transpilation babel react/ --out-dir transpiled/react --verbose && babel helpers/ --out-dir transpiled/helpers --verbose",
+    "transpile:react": "env BABEL_ENV=transpilation babel react/ --out-dir transpiled/react --verbose",
+    "transpile:helpers": "babel helpers/ --out-dir transpiled/helpers --verbose",
```

Comment:
Don't we need the `env BABEL_ENV=transpilation` here  ?

Reply:
Maybe but since it was not present in the first line (in the second command), I think it should not be put here.